lib/rouge/lexers/kotlin.rb

```diff
@@ -87,6 +89,7 @@ class Kotlin < RegexLexer
         rule %r"[0-9](\.[0-9]+)?([eE][+-][0-9]+)?[flFL]?|0[xX][0-9a-fA-F]+[Ll]?", Num
         rule %r/@#{id}/, Name::Decorator
         rule id, Name
+        rule %r'#{plugin_name}|`#{plugin_name}`', Name
```

Comment:
Depending on the approach taken to backticks, it may become unnecessary to include the alternate here.

Reply:
This rule wasn't needed anymore